### PR TITLE
PPLUS-1516: Extend the basic console command with the -m option.

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Ticket: https://spryker.atlassian.net/browse/PPLUS-1516

Change log:
* Adjusted 'ConsoleCommand' with the module filtration option, which specifies which modules should be evaluated. Example `analyze:php:code-compliance -m 'Pyz.ProductStorage'`

### CODE REVIEW

File: src/Upgrader/Console/EvaluateConsole.php [hunk 1]
```diff
@@ -106,4 +141,33 @@ public function execute(InputInterface $input, OutputInterface $output): int
...
     }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     *
+     * @throws \Upgrade\Application\Exception\UpgraderException
+     *
+     * @return array<\Codebase\Application\Dto\ModuleDto>
+     */
+    protected function getModuleList(InputInterface $input): array
```
Comment (alexanderM91) [inline src/Upgrader/Console/EvaluateConsole.php:152]: IMO it shouldn't be part of the console command.
Could be it part of DTO as it's not a business logic but just a data retrieving?

**Comment by alexanderM91** [comment]
@VladislavStrelchenko, 
also could you please update the change log, something like:
- Adjusted 'ConsoleCommand' with the module filtration option, which ......, etc.

File: src/Upgrader/Console/EvaluateConsole.php (hunk 1 above)
Comment (VladislavStrelchenko) [reply src/Upgrader/Console/EvaluateConsole.php]: Thanks 
I created OptionParser for option parsing logic

File: README.md [hunk 2]
```diff
@@ -22,13 +22,17 @@ At this step, the upgrader checks the project against a set of rules. The main g
 
 To evaluate a project, execute the following command from its directory:
 ```bash
-bin/console analyze:php:code-compliance --format=yaml
+bin/console analyze:php:code-compliance
 ```
+The command generates a YAML report at `{project_directory}/reports/`.
+
+Available options:
+- Module filtration option, use it for specifies which modules should be evaluated.
```
Comment (DmytroKlymanSpryker) [inline README.md:30]: I think it is better to write:

-m - module filtration option. It is used to specify the modules for evaluation.

File: README.md [hunk 3]
```diff
@@ -22,13 +22,17 @@ At this step, the upgrader checks the project against a set of rules. The main g
 
 To evaluate a project, execute the following command from its directory:
 ```bash
-bin/console analyze:php:code-compliance --format=yaml
+bin/console analyze:php:code-compliance
```
Comment (alexanderM91) [inline README.md:25]: Why did we remove format?

Comment (VladislavStrelchenko) [reply README.md:25]: We haven't this option in current console and always use `yaml` instead

**Review by DmytroKlymanSpryker — APPROVED** [review]

**Pull request merged**